addons/website_sale/data/mail_template_data.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!--Email template -->
+        <record id="recovery_mail" model="mail.template">
+            <field name="name">Abandoned Cart - Send a Recovery Email</field>
+            <field name="email_from">${(object.user_id.email or '')|safe}</field>
+            <field name="subject">You left items in your cart.</field>
+            <field name="partner_to">${object.partner_id.id}</field>
+            <field name="model_id" ref="sale.model_sale_order"/>
+            <field name="auto_delete" eval="True"/>
+            <field name="lang">${object.partner_id.lang}</field>
+            <field name="body_html" type="xml">
+              <div style="margin:auto;width:100%;">
+                <img src="/web/image/res.company/${user.company_id.id}/logo" style="height: auto; width: 80px; margin-top:30px;" alt="${user.company_id.name}"/>
+                <h1 style="color:#A9A9A9;">THERE'S SOMETHING IN YOUR CART.</h1>
+                <p>Would you like to complete your purchase?</p><br/><br/>
+                % if object.order_line:
+                  % for line in object.order_line:
+                    <hr/>
+                    <table width="100%">
```

Comment:
It means we will generate one table per order line. Can you just bump up the `<table>` tag outside the loop?